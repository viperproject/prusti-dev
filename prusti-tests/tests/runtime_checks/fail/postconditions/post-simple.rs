//@run: 101
use prusti_contracts::*;

fn main() {
    foo(3);
    foo(2);
}

#[trusted]
#[insert_runtime_check]
#[ensures(result % 2 == 0)]
fn foo(x: i32) -> i32 {
    x + 1
}
