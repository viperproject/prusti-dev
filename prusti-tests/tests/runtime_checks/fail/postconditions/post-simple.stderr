thread 'main' panicked at $DIR/post-simple.rs:12:1:
Prusti Runtime Checks: Contract #[ensures(result % 2 == 0)] was violated at runtime
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
